Packages/com.unity.inputsystem/InputSystem/Plugins/InputForUI/InputSystemProvider.cs

```diff
@@ -570,27 +584,31 @@ void RegisterNextPreviousAction()
             m_NextPreviousAction.Enable();
         }
 
-        void UnregisterNextPreviousAction()
+        void UnregisterFixedActions()
         {
+            // The Next/Previous action is not part of the input actions asset
             if (m_NextPreviousAction != null)
             {
                 m_NextPreviousAction.Disable();
                 m_NextPreviousAction = null;
             }
         }
 
-        void RegisterActions()
+        void RegisterActions(ref Configuration cfg)
```

Comment:
Why do we need to pass the config in as args when it's already set as a member variable (and used as such?)

Same question for UnregisterActions, which doesn't use the config at all, but passes it in anyway - are these from merge conflicts perhaps?

Reply:
We do not, guess it was some confusion when I resolved my merge conflicts from develop, I will remove it.

Reply:
Another question, why is the Configuration saved as a field at all?

Reply:
I get the impression the initial design might have considered Configuration to be user provided when it might have made more sense, but right now we could call GetConfiguration() on the fly when needed?

Reply:
Fixed in https://github.com/Unity-Technologies/InputSystem/pull/1903/commits/63c17631cf63b7d68588a639bc181ba35bc1137b

Reply:
> Another question, why is the Configuration saved as a field at all?

Good question! RegisterActions seems to have only two usages, both immediately after setting the m_Cfg with GetDefaultConfiguration, so it seems like GetDefaultConfiguration could just be called at the start of RegisterActions...

Reply:
Resolving this conversation since addressed.